aten/src/ATen/core/dispatch/OperatorEntry.h

```diff
@@ -87,13 +94,30 @@ class OperatorEntry final {
   > kernels_;
   std::mutex kernelsMutex_; // protects kernels_
```

Comment:
just to be paranoid - move mutexes towards the end, so that read path hits the same cache line. Though we still going to do another jump for the std::list or read from the atomic :(